app/models/partners/profile.rb

```diff
@@ -106,7 +106,7 @@ class Profile < Base
     ]
 
     def has_no_social_media?
-      website.blank? && twitter.blank? && facebook.blank? && instagram.blank?
+      (website.blank? || website == "N/A") && (twitter.blank? || twitter == "N/A") && (facebook.blank? || facebook == "N/A") && (instagram.blank? || instagram == "N/A")
```

Comment:
We shouldn't be forced to use "N/A" when an empty string also indicates that there is no value.

Reply:
Ok I will work on that